src/main/java/task/TaskManager.java

```diff
@@ -0,0 +1,50 @@
+package task;
+import java.io.*;
+
+public class TaskManager {
+    private static final String FILE_PATH = "./data/maga.txt";
+
+    public void saveTasks(TaskList taskList) {
+        //create parent directory if it doesn't exist
+        File file = new File(FILE_PATH);
+        File parentDir = file.getParentFile();
+        if (parentDir != null && !parentDir.exists()) {
+            parentDir.mkdirs();
+        }
+
+        //write to file
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(FILE_PATH));
+            for (int i = 0; i < taskList.getTaskCount(); i++) {
+                writer.write(taskList.getTask(i).toString());
+                writer.newLine();
+                writer.flush();
+            }
+        } catch (IOException e) {
+            System.out.println("Error while saving tasks!");
+        }
+    }
+
+    public TaskList loadTasks() {
+        TaskList tasks = new TaskList(100);
+        File file = new File(FILE_PATH);
+
```

Comment:
I like that you are sticking to the style when it comes to indenting comments relative to their position in the code